src/endpoints/checkBackgroundUploadBulk.ts

```diff
@@ -36,8 +36,24 @@ export const checkBackgroundUploadBulk = async (
 
     const response: Response = {};
 
+    const jobs = await workerQueue.getJobs([
+      "completed",
+      "waiting",
+      "active",
+      "delayed",
+      "failed",
+      "paused",
+    ]);
+
     for (const jobId of dto.jobIds) {
-      const job = await workerQueue.getJob(jobId);
+      const index = jobs.findIndex((job) => {
+        return job.id.toString() == jobId.toString();
```

Comment:
nit: we should use `===` for equality

https://stackoverflow.com/questions/27770484/typescript-correct-way-to-do-string-equality